src/components/AddFeed.vue

```diff
@@ -1,110 +1,110 @@
 <template>
-    <Modal @close="$emit('close')">
-        <div id="new-feed" news-add-feed="Navigation.feed">
-            <form ng-submit="Navigation.createFeed(Navigation.feed)"
-                ng-init="Navigation.feed.autoDiscover=true"
-                name="feedform">
-                <fieldset ng-disabled="Navigation.addingFeed" style="padding: 16px">
-                    <input type="text"
-                        :value="feed"
-                        ng-model="Navigation.feed.url"
-                        ng-class="{'ng-invalid':
+	<Modal @close="$emit('close')">
+		<div id="new-feed" news-add-feed="Navigation.feed">
+			<form ng-submit="Navigation.createFeed(Navigation.feed)"
+				ng-init="Navigation.feed.autoDiscover=true"
+				name="feedform">
+				<fieldset ng-disabled="Navigation.addingFeed" style="padding: 16px">
+					<input type="text"
+						:value="feed"
+						ng-model="Navigation.feed.url"
+						ng-class="{'ng-invalid':
                         !Navigation.addingFeed &&
                         Navigation.feedUrlExists(Navigation.feed.url)
                     }"
-                        :placeholder="t('news', 'Web address')"
-                        name="address"
-                        pattern="[^\s]+"
-                        required
-                        autofocus />
-
-                    <p class="error"
-                        ng-show="!Navigation.addingFeed &&
+						:placeholder="t('news', 'Web address')"
+						name="address"
+						pattern="[^\s]+"
+						required
+						autofocus>
+
+					<p class="error"
+						ng-show="!Navigation.addingFeed &&
                         Navigation.feedUrlExists(Navigation.feed.url)">
```

Comment:
this seems inconsistent

Reply:
interesting... it seems like the linter ignores these lines because they are inside of the `"` and I guess are assumed to be intentional spaces in that string definition instead of code style... I can go through and change them all to be consistent though